apps/cnspec/cmd/scan.go

```diff
@@ -397,21 +395,28 @@ This example connects to Microsoft 365 using the PKCS #12 formatted certificate:
 		printReports(report, conf, cmd)
 
 		// handle report sharing
-		var shareReport *bool
+		var shareReport bool
 
 		if viper.IsSet("share-report") {
 			shareReportFlag := viper.GetBool("share-report")
-			shareReport = &shareReportFlag
+			shareReport = shareReportFlag
 		}
 
-		// if we are in an interactive terminal, running in incognito mode, and user responds "yes" then offer report viewer
-		if shareReport == nil && isatty.IsTerminal(os.Stdout.Fd()) && conf.IsIncognito {
-			response := cnspec_components.AskAYesNoQuestion("Do you want to view or share these scan results in a browser using Mondoo's reporting service?")
-			shareReport = &response
+		otherReportOptionsMsg := ""
+		if conf.Output == "compact" {
+			otherReportOptionsMsg += "For detailed CLI output, use `--output full`. "
+		}
+
+		if conf.IsIncognito && shareReport == false {
```

Comment:
Is Incognito the default when cnspec is run without a Platform config?

Reply:
incognito is the default